app/components/atoms/CarouselCircle/CarouselCircle.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react';
+import {CarouselCircle} from './CarouselCircle.component';
+
+export default {
+  title: 'Atoms/CarouselCircle',
+  component: CarouselCircle,
+} as ComponentMeta<typeof CarouselCircle>;
+
+const Template: ComponentStory<typeof CarouselCircle> = (args) => (
+  <CarouselCircle {...args} />
+);
+
+export const Normal = Template.bind({});
```

Comment:
I usually use "Default":
"The default state of this component", rather than "The normal state of this component".